libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/StreamingExtensions/StreamingExtensionRequestHandler.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Protocol;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Integration.AspNet.Core.StreamingExtensions
+{
+    internal class StreamingExtensionRequestHandler : RequestHandler
+    {
+        public StreamingExtensionRequestHandler(BotFrameworkStreamingExtensionsAdapter adapter, IBot bot)
+        {
+            this.StreamingAdapter = adapter;
+            this.Bot = bot;
+        }
+
+        private BotFrameworkStreamingExtensionsAdapter StreamingAdapter { get; set; }
+
+        private IBot Bot { get; set; }
+
+        /// <summary>
+        /// Processes incoming requests and returns the response, if any.
+        /// </summary>
+        /// <param name="request">A ReceiveRequest from the connected channel.</param>
+        /// <returns>A response created by the BotFrameworkStreamingExtensionsAdapter.</returns>
+        public override async Task<Response> ProcessRequestAsync(ReceiveRequest request)
+        {
+            var response = new Response();
+
+            var body = request.ReadBodyAsString().Result;
+
+            if (string.IsNullOrEmpty(body))
+            {
+                response.StatusCode = 400;
+                return response;
+            }
+
+            try
+            {
+                var activity = JsonConvert.DeserializeObject<Activity>(body, SerializationSettings.DefaultDeserializationSettings);
```

Comment:
Again, agree with Joe that the exact same serialization settings are being used.

Reply:
Switched to use the default of the Request class, but will coordinate with Joe to make sure his server is expecting that.

Reply:
👍